Makefile

```diff
@@ -201,13 +208,35 @@ electron-installer-dmg: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$
 TARGETS += \
 	electron-installer-dmg \
 	electron-installer-app-zip
+PUBLISH_AWS_S3 += \
```

Comment:
I don't like how we're duplicating the whole installer target in both the phony targets declarations (`electron-installer-appimage`, `electron-installer-dmg`, etc) and when accumulating them in the `PUBLISH_*` lists.

Maybe we can create variables for each installer and re-use them? For example:

```
INSTALLER_ELECTRON_APPIMAGE = release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
INSTALLER_ELECTRON_DEBIAN = release/out/$(APPLICATION_NAME_LOWERCASE)-electron_$(APPLICATION_VERSION_DEBIAN)_$(TARGET_ARCH_DEBIAN).deb
```

Or something like that?